### DISCUSSION TITLE
SoftBody as a player

### DISCUSSION THREAD

**Discussion by FrancisJasm** [godot-jolt/godot-jolt#963] [Questions]
Hi Mihe,

I was playing with Jolt and I wanted to make a character out of a Softbody sphere...
I notice there is no fonction to apply force to such object,

https://forum.godotengine.org/t/softbody-as-player-apply-forces-move-jump/20177

I've try to link the Softbody with removetransform3d but it's not working.

Is it possible/plan to make such a game possible or Softbody in Godot are more implemented as static object (rope, cloth, cape...)

Thx so much for all the work you did with Jolt, 
Hope it can be merge soon :-) 

Best regards,
Francis

**Comment by mihe** [comment]
> Is it possible/plan to make such a game possible or Softbody in Godot

Honestly, I would say with the `SoftBody3D` interface being what it is currently, this is likely going to be difficult. As you seem to have found out already you can't really do much with them. You can pin some vertices and move those around (or parent them to something) which is what's used for things like capes and whatnot, but besides that there's not a whole lot you can do. Even something like setting the transform of a `SoftBody3D` is somewhat strange, due to quirks on the Godot side of things.

With Jolt not supporting any interactions between `SoftBody3D` and `Area3D` you're also quite limited in those kinds of gameplay mechanics.
